languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -23219,6 +23219,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <pattern>
                     <token regexp="yes">très|trop|un|de|quelque|pour|à|depuis|sous|avec|le
                         <exception scope="previous" regexp="yes">.+'</exception></token>
+                    <token min="0">tout</token>
```

Comment:
thought(unblocking): I don't see any problem with that right now. As you said, let's see the output of the nightly! 🥜